app/Plugin/EnvSource/Config/Schema/schema.xml

```diff
@@ -39,6 +39,8 @@
     </field>
     <field name="duplicate_mode" type="C" size="2" />
     <field name="redirect_on_duplicate" type="C" size="1024" />
+    <field name="sp_type" type="C" size="2" />
+    <field name="other_type" type="C" size="2" />
```

Comment:
I don't think we need `other_type`, see additional comment below.

Reply:
Done